subscribie/blueprints/admin/__init__.py

```diff
@@ -1051,6 +1052,19 @@ def set_reply_to_email():
     )
 
 
+@admin.route("/rename-shop", methods=["GET", "POST"])
+@login_required
+def rename_shop():
+    if request.method == "GET":
+        return render_template("admin/settings/rename_shop.html")
+    elif request.method == "POST":
+        old_name = request.host #this will be active in the real environment
+        #old_name= "example.subscriby.shop" # this will be for local development
+        new_name = request.form.get("new_name", None)
+        subprocess.run(f"./rename-shop.sh {old_name} {new_name}", shell=True)
```

Comment:
@joeltejeda the client side validation is useful for end users experience, but provides no protection. We'll talk together about this soon, in the meantime please read about the difference between client side and server side validation: OWASP website is a good place to look or MDN network